models/classes/runner/config/QtiRunnerConfig.php

```diff
@@ -38,7 +38,14 @@ class QtiRunnerConfig extends ConfigurableService implements RunnerConfig
 
     const OPTION_CONFIG = 'config';
 
+    /**
+     * @deprecated since version 29.2.0, to be removed in 30.0.0. Use QtiRunnerService::TOOL_ITEM_THEME_SWITCHER instead
+     */
     const TOOL_ITEM_THEME_SWITCHER = 'itemThemeSwitcher';
+
+    /**
```

Comment:
Same thing: https://github.com/search?q=org%3Aoat-sa+TOOL_ITEM_THEME_SWITCHER_KEY&type=Code

Reply:
Same comment ;) So we cannot be sure that no other community extension is using that so it's more safe to mark it as deprecated.